### PR TITLE
feat: improve wheel

### LINKED ISSUE #535: Change wheel central icon
Currently it's an arrow pointing to the top. Perhaps follow what was made on the [design](https://www.figma.com/file/sPqEvYKlOqL8L7CC2AzxkN/SEI?type=design&node-id=74-322&mode=design&t=t5bUfmv759NwDDEh-0).

### LINKED ISSUE #536: Disable "Spin the Wheel" button when spinning

### PR DESCRIPTION
**Pull request by joaodiaslobo**
Resolves #535 
Resolves #536

image

### CODE REVIEW

**Review by tiago-bacelar — COMMENTED** [review]
Nice. Since you're working on this though, could you pls do a couple extra things?
- The time delay is implemented inside api.js and that is driving me nuts. Can you please pull it out into Wheel.tsx?
- The token count is only updated after the 5 second delay. Could you also make it update before? Note that you can't run refetchUser, since that would already return any wins immediately, so instead you can just remove 20 tokens manually.
Also, you could have just put isSpinning inside canSpin() :)

File: layout/Attendee/Wheel/Wheel.tsx [hunk 1]
```diff
@@ -73,71 +73,74 @@ function WheelPage() {
...
+          } else if (response.prize.name == "Nada") {
+            updateWheelMessage(
+              <WheelMessage
+                title="You didn't win anything!"
+                description="Better luck next time."
+                onExit={(_) => updateWheelMessage(null)}
+              />
+            );
+          } else {
+            //TODO:: CHANGE THIS MESSAGE
```
Comment (MarioRodrigues10) [inline layout/Attendee/Wheel/Wheel.tsx:119]:
```suggestion
```

File: layout/Attendee/Wheel/Wheel.tsx [hunk 2]
```diff
@@ -196,11 +199,11 @@ function WheelPage() {
...
                   : "bg-gray-400 opacity-50"
               } m-auto mt-10 block h-20 w-64 rounded-full`}
-              disabled={!canSpin() || isSpinning}
+              disabled={!canSpin()}
               onClick={spinTheWheel}
             >
-              <p className="font-ibold font-bold">SPIN THE WHEEL</p>
-              <p className="font-iregular">20 tokens💰</p>
+              <p className="select-none font-ibold font-bold">SPIN THE WHEEL</p>
+              <p className="select-none font-iregular">20 tokens💰</p>
```
Comment (MarioRodrigues10) [inline layout/Attendee/Wheel/Wheel.tsx:206]: This shouldn't be hardcoded but should be grapped from the safira environment, but do it in another PR

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

**Review by MarioRodrigues10 — APPROVED** [review]
:rocket:

**Pull request merged**